src/main/java/codeu/controller/RocketBotHandler.java

```diff
@@ -26,16 +28,26 @@ public void handler(BotRequest request) {
 
     // Convert output and required arguments into Message
     Message message = new Message(
-        UUID.randomUUID(),
+        messageId,
         request.getConversationId(),
         BOT_USER_ID,
         BOT_OUTPUT,
-        Instant.now());
+        messageTime);
 
     // Send message to conversation
     messageStore.addMessage(message);
   }
 
+  /** Accessor class that allows test UUIDs to match, even when randomly assigned. */
```

Comment:
Generally I wouldn't advise making data available publicly only to make tests easier; we really want to test the way this would look to the public caller, not introspecting the internals.

I think this is _okay_, but if you're feeling ambitious, I would instead list the messages in a conversation before and after calling rocketbot, and checking if the added message looks good

Reply:
Looking more, it looks like you're doing this to check that the ID matches in your test. Don't worry about testing that the id created in the class is what was sent, I'd say. You don't care about the internal way things are stored, really. You want to test the content of the message, mostly, and that it's associated with the convo